clang/include/clang/Driver/Options.td

```diff
@@ -1729,8 +1729,6 @@ defm gnu_inline_asm : BoolFOption<"gnu-inline-asm",
           "Disable GNU style inline asm">,
   PosFlag<SetTrue>>;
 
-def fprofile_sample_use : Flag<["-"], "fprofile-sample-use">, Group<f_Group>,
-    Visibility<[ClangOption, CLOption]>;
 def fno_profile_sample_use : Flag<["-"], "fno-profile-sample-use">, Group<f_Group>,
```

Comment:
Why don't also remove -fno-profile-sample-use and just leave -fprofile-sample-use=.
Is there any scenario to use -fno-profile-sample-use?

Reply:
If `-fprofile-sample-use=` is added as a default option, `-fno-profile-sample-use=` can be used to cancel it. For example, it could be used as a debugging tool when you think one TU triggers a compiler bug.